test/components/center_of_mass_test.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:turf/center_of_mass.dart';
+import 'package:turf/meta.dart';
+import 'package:test/test.dart';
+import 'package:turf/src/center_of_mass.dart';
+
+void main () {
```

Comment:
are those all the tests from Turf JS?

If there are more, incl. file testing, then it would be great to  implement those too